README.md

```diff
@@ -109,28 +109,31 @@ When set, it is used to filter out distributions that are not capable of creatin
 There is mostly a 1-to-1 relationship between the DiscoAPI's distributions and [Gradle vendors](https://docs.gradle.org/current/userguide/toolchains.html#sec:vendors).
 The plugin works with the following mapping:
 
-| Gradle JVM Vendor       | Foojay Distribution       |
-|-------------------------|---------------------------|
-| ADOPTIUM                | Temurin                   |
-| ADOPTOPENJDK            | AOJ                       |
-| AMAZON                  | Corretto                  |
-| APPLE                   | -                         |
-| AZUL                    | Zulu                      |
-| BELLSOFT                | Liberica                  |
-| GRAAL_VM                | Graal VM CE 8/11/16/17/19 |
-| HEWLETT_PACKARD         | -                         |
-| IBM                     | Semeru                    |
-| IBM_SEMERU              | Semeru                    |
-| JETBRAINS               | JetBrains                 |
-| MICROSOFT               | Microsoft                 |
-| ORACLE                  | Oracle OpenJDK            |
-| SAP                     | SAP Machine               |
+| Gradle JVM Vendor | Foojay Distribution       |
+|-------------------|---------------------------|
+| ADOPTIUM          | Temurin                   |
+| ADOPTOPENJDK      | AOJ                       |
+| AMAZON            | Corretto                  |
+| APPLE             | -                         |
+| AZUL              | Zulu                      |
+| BELLSOFT          | Liberica                  |
+| GRAAL_VM          | Graal VM CE 8/11/16/17/19 |
+| HEWLETT_PACKARD   | -                         |
+| IBM               | Semeru                    |
+| (IBM_SEMERU)      | Semeru                    |
```

Comment:
I would suggest to go with an asterisk instead of parentheses...🤔 An asterisk indicate more like "read up on this in the small notes"..